plugins/gcc6/README.md

```diff
@@ -0,0 +1,69 @@
+# Notes about GCC version 6.1.0 and later
+
+GCC 6.1 is released with a number of major changes
+[[1](https://gcc.gnu.org/ml/gcc-announce/2016/msg00000.html)]
+[[2](https://gcc.gnu.org/gcc-6/changes.html)]
+. The most notable change is:
+> The `C++` frontend now defaults to `C++14` standard instead of `C++98` it has
+> been defaulting to previously, for compiling older `C++` code that might
+> require either explicitly compiling with selected older `C++` standards,
+> or might require some code adjustment, see
+> [[3](http://gcc.gnu.org/gcc-6/porting_to.html)]
+> for details.
+
+So it is expected that some of MXE packages will fails to build from source
+(FTBFS) with default GCC 6.x options. As a workaround we may add `-std=gnu++11`
+or `-std=gnu++98` into `CXXFLAGS` for building problematic packages. And this
+will not affect the builds with older versions of GCC.
+
+For example, in autotools based projects:
+```
+...
+$(PKG)_CXXFLAGS := -std=gnu++11
+...
+    cd '$(1)' && \
+    CXXFLAGS="$($(PKG)_CXXFLAGS)" \
+    ./configure \
+...
+```
+
+Just after adding this plugin (gcc6) some packages were FTBFS:
```

Comment:
IMHO, s/were FTBFS/failed to build from source/
It was hard to find the meaning :(